databuilder/publisher/neo4j_csv_publisher.py

```diff
@@ -44,6 +44,9 @@
 # Neo4j property name for published tag
 PUBLISHED_TAG_PROPERTY_NAME = 'published_tag'
 
+# Neo4j property name for last updated timestamp
+LAST_UPDATED_EPOCH_MS = 'last_updated_epoch_ms'
```

Comment:
is it the same property name we used in search for last updated timestamp?

Reply:
I don't see `last_updated_epoch_ms` being used in databuilder. However, it's a good point. Let me change it to `publisher_last_updated_epoch_ms`